### ISSUE TITLE
.WithMessage swaps actual/expected in error output

### ISSUE THREAD

**Issue by ahri** [tpierrain/NFluent#159]
```csharp
[Fact]
public void NFlunt_Bug()
{
    Check.ThatCode(() => { throw new Exception("actual"); } )
        .Throws<Exception>()
            .WithMessage("expected");
}
```

Result:
```
NFluent.FluentCheckException

The message of the checked exception is not as expected.
The given exception message:
	["expected"]
The expected value(s):
	["actual"]
   at NFluent.LambdaExceptionCheck`1.WithMessage(String exceptionMessage)
```

**Comment by tpierrain** [comment]
Thanks for your feedback and for using NFluent. A fix has already been made actually ( #114 ). I guess it will be time for us to release a new version of NFluent.

**Comment by ahri** [comment]
Cool, thanks a lot!

**Comment by cmeeren** [comment]
Just want to say that I too experience this - almost a year after it was reported. Did you ever release version with this fix? I suppose it will be part of 2.0?

**Comment by dupdob** [comment]
Yes. Sorry for the delay. Note that you can use the alpha version through Nuget. You need to enable the prerelease check in the nuget gui.

Furthermore, you can also access to the continuous/nightly buid, which is published on myget
https://www.myget.org/F/dupdobnightly/api/v3/index.json
